src/loading.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * An expanded set of loading instructions based on
+ * https://developer.mozilla.org/en-US/docs/Web/Performance/Lazy_loading.
+ *
+ * Only `UNLOAD` is not defined by the "Lazy loading" spec at this time. It's
+ * added here because it supersedes all other loading indstructions in AMP.
```

Comment:
nit: indstructions --> instructions

Reply:
Not sure what to be more amazed by: my ability to make these typos or your ability to spot them!